science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+</figure>
+
+Hierbei wurde ein konstantes Verhältnis des Anteils der Warnenden unter den Neuinfizierten angenommen. Tatsächlich hat sich dieses Verhältnis jedoch über die Zeit entwickelt. Der Unterschied am Ende ist durch den fehlenden Meldeverzug bei den Warnenden (und der Datenbereitstellung über das CWA-Ökosystem) gegenüber dem etablierten Meldesystem zu erklären. Des Weiteren sollten wir bei den Neuinfektionen die jüngeren Altersgruppen nicht berücksichtigen, da diese wegen der Altersbeschränkung (Nutzungszulassung ab 16 Jahren) nicht als Nutzende der CWA angenommen werden. Ebenso hängt der genannte Anteil — und damit die Schätzung — von der Teilungsrate ab, d.h. welcher Anteil derjenigen, die einen Test über die Corona-Warn-App registrieren, ihr Testergebnis erhalten und dieses dann auch mit anderen teilen. Diese Raten sind in der folgenden Abbildung dargestellt.
+
+<figure>
+    <img src="./plot_au_estimate_1_rates_de.png" class="figure-img img-fluid" alt="Teilungsrate und Verhältnis von Warnenden zu Neuinfektionen (unter Berücksichtigung der Altersgrenze)."/>
+    <figcaption aria-hidden="true">Abb. 8: Teilungsrate und Verhältnis von Warnenden zu Neuinfektionen (unter Berücksichtigung der Altersgrenze).</figcaption>
+</figure>
+
+Wenn wir nun annehmen, dass sich das Verhältnis von Warnenden zu Neuinfektionen genauso verhält, wie das Verhältnis der aktiven Nutzenden (im engeren Sinne) zur (relevanten) Bevölkerung unter Berücksichtigung der Teilungsrate (wobei wir für die Bevölkerung 83 Mio. bzw. für die potenziellen Downloadenden 60 Mio. (d.h. [Personen im Alter von über 15 Jahren, die ein Smartphone besitzen](https://de.statista.com/statistik/daten/studie/198959/umfrage/anzahl-der-smartphonenutzer-in-deutschland-seit-2010/)) ansetzen):
+
+<figure>
+    <img src="./fig9_de.png" class="figure-img img-fluid" width=100% alt="Annahme des gleichen Verhaltens von Verhältnis von Warnenden zu Neuinfektionen und Verhältnis der aktiven Nutzenden (im engeren Sinne) zur (relevanten)Bevölkerung unter Berücksichtigung der Teilungsrate"/>
+    <figcaption aria-hidden="true">Abb. 9: Annahme des gleichen Verhaltens von Verhältnis von Warnenden zu Neuinfektionen und Verhältnis der aktiven Nutzenden (im engeren Sinne) zur (relevanten)Bevölkerung unter Berücksichtigung der Teilungsrate</figcaption>
+</figure>
+
+können wir die Anzahl der aktiven Nutzenden bestimmen. Die folgende Abbildung zeigt den zeitlichen Verlauf dieser Schätzung.
+
+<figure>
+    <img src="./plot_au_estimate_1_estimate_de.png" class="figure-img img-fluid" alt="Schätzung der aktiven Nutzenden aus der Anzahl der Warnenden."/>
+    <figcaption aria-hidden="true">Abb. 10: Schätzung der aktiven Nutzenden aus der Anzahl der Warnenden.</figcaption>
+</figure>
+
+Für die letzten 30 Tage (bei Nichtberücksichtigung der letzten 5 Tage, aufgrund des Meldeverzugs bei den Neuinfektionen: 25. Januar 2022 – 23. Februar 2022) erhalten wir damit 27 Mio. (KI: 21 Mio. – 33,3 Mio.).
```

Comment:
Please specify what the "letzten 5 Tage" are. From the day when the post is published or written?

Reply:
Hi Tim,

in this case the data is from 01.03.2022. The last five days are not included and then the period of the last 30 days is considered. This is how the specified period from January 25, 2022 to February 23, 2022 is calculated.

Best regards
Tobias

Reply:
Hi Tobias,

thanks for your answer! Would it make sense to add this to the blog? I would provide a PR for this! 

Appreciate your feedback!

Reply:
Hi Tim, 

thanks for your kind offer, but we will add some edits in the near future and explain these numbers for a better understanding. 

Best regards,
Tobias

Reply:
Okay, thanks!